charts/victoria-metrics-common/templates/_helpers.tpl

```diff
@@ -0,0 +1,59 @@
+{{- /* Expand the name of the chart. */ -}}
+{{- define "vm.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{- /*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/ -}}
+{{- define "vm.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{- /* Create chart name and version as used by the chart label. */ -}}
+{{- define "vm.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{- /* Create the name of the service account to use */ -}}
+{{- define "vm.sa" -}}
```

Comment:
```suggestion
{{- define "vm.sa.name" -}}
```

Let's add a `name` suffix as otherwise it seems like this template will render full SA config.

Reply:
thanks for review, will recreate this PR later, after making these templates more generic